lib/app/utils/hl7_validator.rb

```diff
@@ -33,7 +38,7 @@ def validate(resource, fhir_models_klass, profile_url = nil)
     private
 
     def issues_by_severity(issues, severity)
-      issues.reject { |i| i.code == 'code-invalid' }
+      issues.reject { |i| i.code == 'code-invalid' || DETAILS_FILTER.include?(i.details.text) }
```

Comment:
Would it make sense to have a 'sanitize_issues' method that just does these 2 things?    Then in the method that calls this do something like `sanitized_issues = sanitize_issues(outcome.issue)` followed by a bunch of `xyz = issues_by_severity(sanitized_issues, 'xyz')`.  I just worry a little that if you need to go through this method in order to throw out the ones we want to hide, but if for some reason someone wants all issues (not just fatal, etc), they won't realize that they need to do copy in this `reject` line first.

Reply:
Based on further discussion we are going to change this anyhow, so disregard this comment.